src/components/account-deletions-report.tsx

```diff
@@ -0,0 +1,84 @@
+import { useContext, useRef } from "preact/hooks";
+import { ascending, descending, flatGroup } from "d3-array";
+import * as Plot from "@observablehq/plot";
+import useRegistrationData from "../hooks/use-registration-data";
+import useElementWidth from "../hooks/use-element-width";
+import { ReportFilterContext } from "../contexts/report-filter-context";
+import { formatAsPercent, yearMonthDayFormat } from "../formats";
+import PlotComponent from "./plot";
+import Table from "./table";
+import type { ProcessedResult } from "../models/daily-registrations-report-data";
+import type { TableData } from "./table";
+import { format } from "d3-format";
+
+interface ProcessedFormattedData {
+  date: Date;
+  value: number;
+}
+
+function plot({ data, width }: { data: ProcessedFormattedData[]; width?: number }): HTMLElement {
+  return Plot.plot({
+    color: {
+      type: "ordinal",
+      scheme: "Tableau10",
+    },
+    marks: [Plot.ruleY([0]), Plot.line(data, { x: "date", y: "value" })],
+    width,
+    y: {
+      domain: [0, 0.1],
+    },
```

Comment:
let's add in `tickFormat` here and pass in something to format these as percentages?
```js
import { format  } from 'd3-format'
```

```suggestion
    y: {
      domain: [0, 0.1],
      tickFormat: format(".2%"),
    },
```

(I would move this to a constant and put it in `formats.ts` but the gist is this)

Reply:
Aha, I was wondering how to make the percentages a little nicer looking in the plot.

Reply:
> Aha, I was wondering how to make the percentages a little nicer looking in the plot.

Added in cd99a88.